t/proxyunits.t

```diff
@@ -444,14 +504,36 @@ check_version($ps);
     print $be "END\r\n";
     is(scalar <$ps>, "END\r\n", "ltrimkey END");
 
-    # token(n) fetch
-    # token(n, "replacement")
-    # token(n, "") removal
-    # ntokens()
-    # command() integer
```

Comment:
@dormando 

What's is this test scenario?

Reply:
assert `r:command() == mcp.CMD_GET` etc, probably?